rules/do-you-know-how-to-claim-expense-reimbursements-2/rule.md

```diff
@@ -116,7 +83,8 @@ Note - if not already done, make sure purchase approval is forwarded to the Acco
 
 
 
-<font color="#333333"> <strong></strong> </font>1. Review the Xero Expense claim weekly
+<font color="#333333"> <strong></strong> </font>
+1. Review the Xero Expense claims weekly on Thursday's
```

Comment:
I don't think Thursdays should have an apostrophe.

Also I think the wording "Review the Xero Expense claims every Thursday" is better, what do you think?

Reply:
Yes, agreed to both